extensions/amp-render/1.0/test/validator-amp-render.out

```diff
@@ -0,0 +1,143 @@
+|
+|      <!-- Valid: amp-render with binding="refresh" attribute -->
+|      <amp-render
+|        width="10"
+|        height="10"
+|        src="https://data.com/articles.json?ref=CANONICAL_URL"
+|        binding="refresh"
+|      >
+|      </amp-render>
+|
+|      <!-- Valid: amp-render with binding="no" attribute -->
+|      <amp-render
+|        width="10"
+|        height="10"
+|        src="https://data.com/articles.json?ref=CANONICAL_URL"
+|        binding="no"
+|      >
+|      </amp-render>
+|
+|      <!-- Valid: amp-render with binding="always" attribute -->
+|      <amp-render
+|        width="10"
+|        height="10"
+|        src="https://data.com/articles.json?ref=CANONICAL_URL"
+|        binding="always"
+|      >
+|      </amp-render>
+|
+|      <!-- Valid: amp-render with credentials="include" -->
+|      <amp-render
+|        width="10"
+|        height="10"
+|        src="amp-script:scriptFn.dataFn"
+|        credentials="include"
+|      >
+|      </amp-render>
+|
+|      <!-- Valid: amp-render with credentials="omit" -->
+|      <amp-render
+|        width="10"
+|        height="10"
+|        src="https://data.com/articles.json?ref=CANONICAL_URL"
+|        credentials="omit"
+|      >
+|      </amp-render>
+|
+|      <!-- Valid: amp-render with xssi-prefix attribute -->
+|      <amp-render
+|        width="10"
+|        height="10"
+|        src="https://data.com/articles.json?ref=CANONICAL_URL"
+|        xssi-prefix="({["
+|      >
+|      </amp-render>
+|
+|      <!-- Valid: amp-render with key attribute -->
+|      <amp-render
+|        width="10"
+|        height="10"
+|        src="https://data.com/articles.json?ref=CANONICAL_URL"
+|        key="name"
+|      >
+|      </amp-render>
+|
+|      <!-- Invalid: missing at least src, [src] or data-amp-bind-src. -->
+|      <amp-render width="10" height="10"> </amp-render>
+>>     ^~~~~~~~~
+amp-render/1.0/test/validator-amp-render.html:118:4 The tag 'amp-render' is missing a mandatory attribute - pick at least one of ['src','[src]','data-amp-bind-src']. (see https://amp.dev/documentation/components/amp-render/)
+|
+|      <!-- Invalid: container layout -->
+|      <amp-render
+>>     ^~~~~~~~~
+amp-render/1.0/test/validator-amp-render.html:121:4 The specified layout 'CONTAINER' is not supported by tag 'amp-render'. (see https://amp.dev/documentation/components/amp-render/)
+|        layout="container"
+|        src="https://data.com/articles.json?ref=CANONICAL_URL"
+|      >
+|      </amp-render>
+|
+|      <!-- Invalid: invalid binding attribute -->
+|      <amp-render
+>>     ^~~~~~~~~
+amp-render/1.0/test/validator-amp-render.html:128:4 The attribute 'binding' in tag 'amp-render' is set to the invalid value 'foobar'. (see https://amp.dev/documentation/components/amp-render/)
+|        width="10"
+|        height="10"
+|        src="https://data.com/articles.json?ref=CANONICAL_URL"
+|        binding="foobar"
+|      >
+|      </amp-render>
+|    </body>
+|  </html>
```

Comment:
nit: newline here I think

Reply:
This file is autogenerated, so don't want to mess with the output.